.github/workflows/build_scoped_rector.yaml

```diff
@@ -37,8 +37,7 @@ jobs:
 
             # early downgrade individual functions and files of symfony Attribute classes
             -   run: bin/rector process src/functions -c build/config/config-downgrade.php --ansi
-            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute/Autoconfigure.php -c build/config/config-downgrade.php --ansi
-            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute/AutoconfigureTag.php -c build/config/config-downgrade.php --ansi
+            -   run: bin/rector process vendor/symfony/dependency-injection/Attribute -c build/config/config-downgrade.php --ansi
```

Comment:
@TomasVotruba previuos one by one run on `Autoconfigure.php` and `AutoconfigureTag.php` is on purpose as not works to touch both files.

Reply:
Ok, it seems reverted at latest main 

https://github.com/rectorphp/rector-src/blob/96dbc6956249779658a30c7e1a05a5ec396e94d7/.github/workflows/build_scoped_rector.yaml#L40-L41

Reply:
I tried it because this should work either way. What is the desired change of those 2 files?

Reply:
that should remove the Attribute, and by use the directory, it skipped them.

Reply:
That's weird, I'll look into it.

Also, the attribute is interpreted as a comment in older PHP, so this change is probably not needed. The `#[Attribute]` can be left there as a comment
https://3v4l.org/7k941

Reply:
previously, it error:

```bash
Uncaught TypeError: PhpParser\Internal\TokenStream::getIndentationBefore(): Return value must be of type int, null returned
```
see https://github.com/rectorphp/rector-src/pull/133#issuecomment-851972838

Looking at latest main, it seems the : 

```php
new DowngradeAttributeToAnnotation(
                     'Attribute',
                     'Attribute'
),
```

it removed. See https://github.com/rectorphp/rector-src/commit/02f9524e3414e9f65c38611dfc38ccefb5257695#diff-c195c143f1dc6e27a371a830fc13d5d65d089c21c89483c9766d650b7ba46484R21

Reply:
The real problem seems as the DowngradeAnnotation not run, the `DowngradeParameterTypeWideningRector` not executed, that make fatal error.

Reply:
I created PR for by directory at https://github.com/rectorphp/rector-src/pull/367